src/main/java/training/Elbow/ElbowCommandBuilder.java

```diff
@@ -0,0 +1,55 @@
+package training.Elbow;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+
+
+public class ElbowCommandBuilder {
+
+	private final Elbow elbow;
+
+	public ElbowCommandBuilder(Elbow elbow) {
+		this.elbow = elbow;
+	}
+
+
+	public Command moveElbowToAngle(Rotation2d targetAngle) {
+		return new FunctionalCommand(
+			() -> {},
+			() -> elbow.setTargetAngle(targetAngle),
+			(interrupted) -> {},
+			() -> elbow.isAtAngle(targetAngle),
+			elbow
+		);
+	}
+
+	public Command moveElbowUpAndDown() {
+		return new SequentialCommandGroup(
+			upElbow(),
+			new WaitCommand(ElbowConstants.DEFAULT_TIME_IN_AIR_ELBOW_SECONDS),
+			downElbow()
+		);
+	}
+
+	public Command moveElbowUpAndDown(double timeInAirSeconds) { // i know there is a Time object in wpilib, i'll change that
```

Comment:
dfuck why the hell there is time object lol

Reply:
in the units library

Reply:
deletes the comment and marked as resolved.